packages/reactive-element/src/reactive-element.ts

```diff
@@ -1048,7 +1048,11 @@ export abstract class ReactiveElement
       }
     }
     if (elementProperties.size > 0) {
-      queueMicrotask(() => {
+      // This has to run in a microtask so that we go after field
+      // initialization in subclasses. This microtask can be removed when both
+      // standard decorator addInitializer() runs after field initialization
+      // and experimental decorators are no longer supporter, or
```

Comment:
trailing sentence, but I assume the `or` relates to: https://github.com/microsoft/TypeScript/issues/55669